src/main/java/com/nike/cerberus/command/core/CreateWafLoggingCommand.java

```diff
@@ -45,6 +45,13 @@ public boolean isSkipStackCreation() {
         return skipStackCreation;
     }
 
+    @Parameter(names = {"--s3-prefix"}, description = "The prefix of WAF logs in S3 bucket.")
+    private String s3Prefix = "firehose/";
```

Comment:
Does this mean the s3Prefix will always be `firehose/`?

Reply:
It defaults to firehose/ which is the current prefix https://github.com/Nike-Inc/cerberus-lifecycle-cli/blob/master/src/main/resources/cloudformation/waf-logging.yaml#L31
Just trying to make it backwards compatible

Reply:
So if a prefix is provided `firehose/` would be overwritten?

Reply:
Correct